engines/mysql.py

```diff
@@ -47,11 +47,8 @@ def insert_data_from_file(self, filename):
         insert."""
         self.get_cursor()
         ct = len([True for c in self.table.columns if c[1][0][:3] == "ct-"]) != 0
-        if (self.table.cleanup.function == no_cleanup
-            and not self.table.fixed_width
-            and not ct
-            and (not hasattr(self.table, "do_not_bulk_insert") or not self.table.do_not_bulk_insert)
-            ):
+        if self.table.cleanup.function == no_cleanup and not self.table.fixed_width and not ct and (
+                    not hasattr(self.table, "do_not_bulk_insert") or not self.table.do_not_bulk_insert):
```

Comment:
I kind of prefer the one condition per line hear from readability, but if we're going to combine multiple conditions on a single line then I think we definitely want to keep the last `(` with the second line to avoid confusion about the fact that those two conditions are grouped.